recipes/belt/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "BELT" %}
+{% set version = "BELTv1.0" %}
```

Please use a numeric version
```suggestion
{% set version = "1.0" %}
```